src/libraries/Microsoft.NET.WebAssembly.Threading/src/Microsoft.NET.WebAssembly.Threading.proj

```diff
@@ -0,0 +1,47 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+  <PropertyGroup>
+    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <PackageDescription>Exposes Threading APIs for WebAssembly projects</PackageDescription>
+    <PackageId>$(MSBuildProjectName)</PackageId>
+    <IncludeBuildOutput>true</IncludeBuildOutput>
+    <DebugSymbols>false</DebugSymbols>
+    <DebugType>none</DebugType>
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
+    <IsPackable>true</IsPackable>
+    <!-- XXX FIXME: NU5128 suppresses a really scary message about missing net7.0 dependencies -->
+    <NoWarn>$(NoWarn);NU5128;NU5131</NoWarn>
+    <!-- Place assemblies in the nuget ref folder -->
+    <BuildOutputTargetFolder>ref</BuildOutputTargetFolder>
+    <!-- XXX FIXME: the rest of these props affect the NoTargets SDK in some way.  I want to be able to reference this project from another .csproj -->
+    <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+    <DisableImplicitFrameworkReferences>false</DisableImplicitFrameworkReferences>
+    <GenerateDependencyFile>true</GenerateDependencyFile>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Threading.Thread.WebAssembly.Threading\ref\System.Threading.Thread.WebAssembly.Threading.csproj" Pack="true" PrivateAssets="all" Private="true" IncludeReferenceAssemblyInPackage="true" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Threading.ThreadPool.WebAssembly.Threading\ref\System.Threading.ThreadPool.WebAssembly.Threading.csproj" Pack="true" PrivateAssets="all" Private="true" IncludeReferenceAssemblyInPackage="true" />
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Diagnostics.Tracing.WebAssembly.PerfTracing\ref\System.Diagnostics.Tracing.WebAssembly.PerfTracing.csproj" Pack="true" PrivateAssets="all" Private="true" IncludeReferenceAssemblyInPackage="true" />
+  </ItemGroup>
+
+  <!-- https://github.com/nuget/home/issues/3891 add the references .dlls to the package, and don't add the to the nuspec -->
+  <Target Name="CopyProjectReferencesToPackage" DependsOnTargets="BuildOnlySettings;ResolveReferences">
+    <ItemGroup>
+      <BuildOutputInPackage Include="@(ReferenceCopyLocalPaths->WithMetadataValue('ReferenceSourceTarget', 'ProjectReference')->WithMetadataValue('PrivateAssets', 'All'))" />
+    </ItemGroup>
+  </Target>
```

Comment:
Hmm, why do you need this extra target? https://github.com/lambdageek/runtime/blob/eed9c390017a5a52c2b0ce7a6be5deae0e8660ac/eng/packaging.targets#L233-L258 should already do all that for you.

Reply:
Yea I think I can delete this. It's leftover from trying to get both `PackageReference`  and `ProjectReference` working.   Since I gave up on `ProjectReference`, I don't think this is needed anymore.